confirmation/views.py

```diff
@@ -20,14 +20,12 @@
 def confirm(request, confirmation_key):
     # type: (HttpRequest, str) -> HttpResponse
     obj = Confirmation.objects.confirm(confirmation_key)
-    ctx = {'confirmed': False}  # type: Dict[str, Any]
-    templates = [
-        'confirmation/confirm.html',
-    ]
+    context = {'confirmed': False}  # type: Dict[str, Any]
+    template = 'confirmation/confirm.html'
     if obj:
-        ctx = {
+        context = {
             'key': confirmation_key,
             'full_name': request.GET.get("full_name", None),
         }
-        templates = ['confirmation/confirm_preregistrationuser.html']
-    return render(request, templates, context=ctx)
+        template = 'confirmation/confirm_preregistrationuser.html'
+    return render(request, template, context=context)
```

Comment:
I think this might actually read better with 2 code paths that each call `render` rather than setting up the `context` and `template` variables.